tests/slenkins/slenkins_node.pm

```diff
@@ -39,7 +39,8 @@ sub run {
         set_var("NETWORK$i", $control_settings->{"NETWORK$i"}) if $control_settings->{"NETWORK$i"};
     }
 
-    type_string("rcnetwork stop\n");
+    # we have to completely stop wicked - bsc#981651
+    type_string("rcwickedd stop\n");
```

Comment:
the soft failure would appear in unrelated slenkins tests, I don't think we want it.